src/types.ts

```diff
@@ -149,6 +149,7 @@ export declare namespace Lunch {
         // TODO: Why doesn't ConstructorParameters<THREE.WebGLRenderer> work here?
         rendererArguments?: object
         rendererProperties?: Partial<THREE.WebGLRenderer>
+        sizePolicy?: string
```

Comment:
Instead of `string`, could you add a new type `SizePolicy` in the Lunch namespace and set it to:

`type SizePolicy = 'container' | 'full'`

then update this `sizePolicy` property to use that new type?